plugins/fpc/fu-fpc-device.c

```diff
@@ -34,6 +35,21 @@
 #define FPC_DEVICE_NORMAL_MODE_CLASS 0xFF
 #define FPC_DEVICE_NORMAL_MODE_PORT  0xFF
 
+#define FPC_FF2_BLOCK_META_TYPE	       0xcd
+#define FPC_FF2_META_BINMAGIC	       "FPC0001"
+#define FPC_FF2_BINMAGIC_LEN	       7
+#define FPC_FF2_BLOCK_DATA_DIR_OUT     0x0
+#define FPC_FF2_BLOCK_DATA_DIR_IN      0x1
+#define FPC_FF2_BLOCK_DATA_END_LEN     1
+#define FPC_FF2_HDR_OFFSET	       0
```

Comment:
can you describe any more of the header? e.g. does it contain the firmware version, or the device vid/pid perhaps?